### ISSUE TITLE
Add ability to specify onset of symptoms in HPO list.

### ISSUE THREAD

**Issue by pnrobinson** [monarch-initiative/HpoCaseAnnotator#46]
This would make for better Phenopackets.

**Comment by ielis** [comment]
Yes this is a good idea. However, I don't see it just as a quick fix but as an issue that would require some work. We would need to extend our `DiseaseCase` data model which is not a problem.

I think that we should also work on this in `HpoTextMining`, which is the place where the actual phenotypic information is being curated.
